velox/docs/functions/iceberg/coverage.rst

```diff
@@ -0,0 +1,20 @@
+.. raw:: html
+
+    <style>
+    div.body {max-width: 1300px;}
+    table.coverage th {background-color: lightblue; text-align: center;}
+    table.coverage td:nth-child(6) {background-color: lightblue;}
+    table.coverage td:nth-child(8) {background-color: lightblue;}
+    table.coverage tr:nth-child(1) td:nth-child(1) {background-color: #6BA81E;}
+    </style>
+
+.. table::
+    :widths: auto
+    :class: coverage
+
+    =========================================  =========================================  =========================================  =========================================
+    Scalar Functions
+    ==========================================================================================================================================================================
+    :iceberg:func:`bucket`                     truncate                                   years                                      months
```

Comment:
I place `:iceberg:func:`bucket` here to make sure the framework can work well. CC @rui-mo

Reply:
<img width="324" alt="Screenshot 2025-06-04 at 16 53 42" src="https://github.com/user-attachments/assets/7dded449-2742-4386-adf8-32cf30fd0408" />

Is the title name and coverage (truncate etc.) correct?